misc-tools/force-passwords.in

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/python3
+
+import subprocess
+
+webappdir = '@domserver_webappdir@'
+etcdir = '@domserver_etcdir@'
+
+with open(f'{etcdir}/restapi.secret', 'r') as f:
+    while True:
+        line = f.readline()
+        if line.startswith('#'):
+            continue
+        tokens = line.split()
+        if len(tokens) == 4:
```

Comment:
Nitpick: maybe also check that token[0]=='default'?

Reply:
Done